{"uid":"7d25ccd86f964a85","name":"SyncManager sync throws error when documents sync fails","historyId":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager:.frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager#SyncManager sync throws error when documents sync fails","time":{"start":1752052431000,"stop":1752052431042,"duration":42},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false,"attachmentStep":false,"attachmentsCount":0},"afterStages":[],"labels":[{"name":"resultFormat","value":"junit"},{"name":"suite","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"testClass","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"},{"name":"package","value":".frontend.packages.internal.catalyst_voices_services.test.src.sync.sync_manager"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":2,"unknown":0,"total":2},"items":[{"uid":"b3bfeb4af607f39e","reportUrl":"https://input-output-hk.github.io/catalyst-voices/allure-action/fix_f14rc6-bugs-fixes/test-report/16163374444_1752049432615/#testresult/b3bfeb4af607f39e","status":"passed","time":{"start":1752047711000,"stop":1752047711051,"duration":51}}]},"tags":[]},"source":"7d25ccd86f964a85.json","parameterValues":[]}